content/guides/developer-guides/delfin/delfin-kubernetes-guide/_index.md

```diff
@@ -0,0 +1,157 @@
+---
+title: delfin
+description: "delfin deployment on kubernetes guide."
+tags: ["installation guide", "delfin"] 
+---
+
+## Delfin deployment on kubernetes cluster
+Here are the steps to deploy delfin on a kubernetes  cluster 
```

Comment:
Please mention some more details on "Delfin in Kubernetes" 
(I mean, what it the deployment of delfin in kubernetes means to the user? any known limitations, what will work, what user expect etc)

Reply:
Mentioned